setup.cfg

```diff
@@ -1,7 +1,7 @@
 [metadata]
 name = jdaviz
 author = JDADF Developers
-author_email = nearl@stsci.edu
+author_email = rosteen@stsci.edu
```

Comment:
Belated review: We can also use `help`, which goes to Help Desk. But this is good for now.